src/plugins/intel_cpu/src/nodes/batch_to_space.cpp

```diff
@@ -156,6 +156,7 @@ void BatchToSpace::batchToSpaceKernel() {
     size_t channels = (inShape5D[1] / blockSize);
     channels = channels == 0 ? 1 : channels;
     const size_t workAmount = inShape5D[0] * channels;
+    OPENVINO_ASSERT(workAmount > 0, "work amount is 0!");
```

Comment:
1. Why `workAmount` equal `zero` is a problem at all? `splitter` function call below seems to have correct handling logic for such cases.
2. Such error message doesn't help to localize the problem. It should at least contain operation name and type. See https://github.com/openvinotoolkit/openvino/blob/58467282f6fb29590cac34d27f02180b44e79c1d/src/plugins/intel_cpu/src/nodes/batch_to_space.cpp#L61 as an example

Reply:
1. Linter's concern here is mostly about `(end - 1)` on line 167. splitter will work correctly, but (end - 1) in case of zero size will be -1.
2. Thanks, changed the error message

Reply:
1. Ok, agree.
2. So now it has type, but still lacks name. Please use errorPrefix as other THROW routines inside the file